tests/cert_without_extensions.rs

```diff
@@ -0,0 +1,13 @@
+use std::convert::TryFrom;
```

Comment:
Please add the webpki License to the top of the file.

Instead of `std`, use `core` whenever possible.